### PR TITLE
backup/restore crd bugfixes

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#1124 #1125 

### Restore
Screen Shot 2020-09-16 at 9 48 28 AM
Screen Shot 2020-09-16 at 9 48 34 AM
Screen Shot 2020-09-16 at 9 48 40 AM
Screen Shot 2020-09-16 at 9 48 59 AM
Screen Shot 2020-09-16 at 9 36 09 AM
Screen Shot 2020-09-16 at 9 36 44 AM
Screen Shot 2020-09-16 at 9 36 17 AM
Screen Shot 2020-09-16 at 9 36 21 AM

### Backup
Screen Shot 2020-09-16 at 10 02 36 AM
Screen Shot 2020-09-16 at 10 03 23 AM

### CODE REVIEW

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -142,15 +142,24 @@ backupRestoreOperator:
...
   resourceSetName: Resource Set
   schedule: 
     label: Schedule
     placeholder: e.g. @midnight or 0 0 * * *
+    options: 
+      disabled: One Time Backup
+      enabled: Schedule Backups
+  restoreFrom:
+    existing: An existing backup config
+    default: The deault storage target
```
Comment (mrajashree) [inline assets/translations/en-us.yaml:161]: typo: deault -> default

**Comment by mrajashree** [comment]
For Restore page, can we change the tooltip for "Encryption Config Secret" to:
```
If the backup was performed with encryption enabled, a secret containing the same encryption-provider-config should be used during restore
```

**Comment by mantis-toboggan-md** [comment]
Screen Shot 2020-09-16 at 10 33 12 AM

**Comment by mrajashree** [comment]
@deniseschannon presenting `Schedule` field only when the `Schedule Backups` option is selected makes it much more clear 👍 
can we rename the radio button from `Schedule Backups` to `Recurring Backups`?

**Comment by vincent99** [comment]
This looks much better now.

"One Time" -> "One-Time"

**Comment by mantis-toboggan-md** [comment]
Updated schedule options to 'One-Time Backup' and 'Recurring Backups'

**Comment by deniseschannon** [comment]
Can we make sure to have prune disabled by default on restore?

**Comment by mantis-toboggan-md** [comment]
According to @mrajashree prune should be enabled by default https://github.com/rancher/dashboard/issues/1125#issuecomment-691425362

**Comment by deniseschannon** [comment]
Since delete timeout is only used during prune, delete timeout should only be a visible input if prune is enabled. 

New Tooltip text:
Prune
```
Delete all resources that are not in the backup, but match the ResourceSet used during backup. (Recommended)
```

Delete Timeout
```
Amount of time to wait while deleting a resource before editing the resource to remove finalizers and attempt to delete again.
```

**Comment by mantis-toboggan-md** [comment]
Screen Shot 2020-09-16 at 3 58 48 PM
Screen Shot 2020-09-16 at 3 58 57 PM

**Comment by vincent99** [comment]
fyi I changed the words on those a bit..

**Pull request merged**